### ISSUE TITLE
Error: Invoice line item prefix check too strict

### ISSUE THREAD

**Issue by kiljacken** [wyyerd/stripe-rs#64]
Hi, we recently encountered issues processing a subscription renewal due to the new typed IDs.

The ID of the received line-item is `sli_321c032d98fd96` which does not match the typical `ii_` prefix, presumably due to it being a subscription. We've rolled back to an older version of our webhook for now, but would love to know if there's a way to fix the issue.

The error message produced:
```
Error("invalid `InvoiceLineItemId`, expected id to start with \"ii_\"", line: 172, column: 3)
```

Thanks.

Slightly redacted webhook payload: [gist](https://gist.github.com/kiljacken/872ba427877a0fda0daacc036e03ff47)

**Comment by kestred** [comment]
Ah.   Yes, this is a correctness issue and should be fairly straightforward to fix--  was this using a `0.10.x` version, or a prior version?

**Comment by kiljacken** [comment]
This is trying out 0.10. We've been running on a fork previously, and
decided to give the new version a try and test if it fixed the issue we had.

**Comment by kestred** [comment]
This was published as part of `v0.10.2`

**Comment by kiljacken** [comment]
Awesome! Thank you very much
